scripts/scripts.js

```diff
@@ -310,6 +310,26 @@ export function getWindowSize() {
     height: windowHeight,
   };
 }
+
+export function addTopSpacingStyleToFirstMatchingSection(main) {
```

Comment:
I think this can be invoked from `decorateSections` which deals with such section decoration logic.

Reply:
But I need the block decoration to be done in order for the sections to have the `*-container` classes, which get added by the `decorateBlock` function